src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -3,39 +3,74 @@ title: Usage
 order: 1
 ---
 
-import { Paragraph } from '@ecl/website-components';
+import { Link, Paragraph } from '@ecl/website-components';
 
 <Paragraph size="lead">
   Use a button to trigger and carry out an action on a page.
 </Paragraph>
 
-## When to use
-
-The ECL has five buttons with different visual designs to make a distinction between the kind of action the button will carry out.
+## Anatomy
 
-| Button name           | Visual style                                                                                            | Usage                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
-| Call-to-action button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | for the most important action on the page like "Apply for funding" or "Watch live streaming". |
-| Primary button        | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, to initiate a search query or to apply a filter.                |
-| Secondary button      | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | or secondary actions.                                                                         |
-| Ghost button          | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, for instance to "cancel" a form.         |
-| Search button         | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                      |
+| Name  | Mandatory |
+| ----- | :-------: |
+| Label |     X     |
+| Icon  |           |
```

Comment:
```suggestion
| Icon  |           | 
```
add an optional column so we can choose icon as opitonal

Reply:
What is the meaning of your suggested change? 
Do you want to make the icon mandatory? it's not.

Reply:
****I was thinking to add a column to say "Optional" so the user knows to use of icons is optional. Not mandatory.